package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
> These can also happen in micro bumps, when the interface is not changed, but simply linter rules are added in a backwards compatible manner.

That would not be backwards-compatible. I'm wondering if you've actually read the [semantic versioning spec][spec]?

I mean honestly if this is _really_ a concern, we can enable [Greenkeeper](https://greenkeeper.io/). That's probably a good idea anyway, honestly. This whole argument may be mostly irrelevant too since upstream looks [pretty inactive][upstream]. I looked for a better alternative on npm but didn't really find any.

 [spec]: http://semver.org/
 [greenkeeper]: https://greenkeeper.io/
 [upstream]: https://github.com/zaach/jsonlint